clang/test/Preprocessor/init-riscv.c

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: riscv-registered-target
```

Comment:
Does this really require riscv-register-target? It doesn't run backend code generation. None of the riscv- tests in the Preprocessor directory have this.